courses/machine_learning/text_classification/automl_for_text_classification.ipynb

```diff
@@ -0,0 +1,499 @@
+{
```

Comment:
<p>You have a comment here about repeatable splits for train/eval/test but it doesn't seem to show up in the queries? Also (and instead), might be mentioning that you can pass to AutoML the TRAIN/VALID/TEST catagory for each training example or just let AutoML figure it out. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/562/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>Done!</p>